_posts/Ibbi2010/2017-05-22-Ibbi2010-turtlehack.md

```diff
@@ -0,0 +1,27 @@
+---
+layout: post
+author: Ibbi2010
+title: "Ibbi2010's turtle exercise"
+---
+
+My initial experience with the trinket was almost no experience. It took me a little while to explore what a turtle could do. Once that happened, I wanted to try as many possibilities as I could. But keeping in view my limited knowledge/vocabulary about python, I wanted to polish what I had already learnt. which was mainly creating shapes by using left, right, angles with varying degrees and “forward” codes. 
+I don’t know from where  did this idea of “NY” skyline came , but as soon as it did, I knew this would be perfect for practicing the codes I knew.  The first thing I did, was to get a print out of  the skyline silhouette and from thereon pretty much experimented with the angles and distances.
+The biggest problems I faced were, basically mirroring both the sides of buildings, using (“go.to”) code to find out various locations on screen and making the triangle at the top of ESB. I struggled making the right side of ESB straight , which tilted after I made the top triangle. I made it work by adjusting the angles. This is how,
+
+```
+nael.forward(20)
+nael.left(95)
+nael.forward(90)
+nael.right(190)
+nael.forward(89)
+nael.left(95)
+nael.forward(20)
+```
+
+But then there were “A LOT” of adjustments.
+Also I think there is a huge room for improvement to make “Chryslers” dome a little better. By this time,I had become tired and bored with the same set of repetitive codes. I did like the end result though. what do you think? 
```

Comment:
I think the end result is great!  It's a very precise result, shows good persistence getting there, and you've competently used many of Turtle's features.  Bravo!